package.json

```diff
@@ -88,10 +74,10 @@
   ],
   "repository": {
     "type": "git",
-    "url": "http://github.com/offirgolan/ember-cp-validations.git"
+    "url": "https://github.com/qonto/ember-cp-validations.git"
   },
   "license": "MIT",
-  "author": "Offir Golan <offirgolan@gmail.com>",
+  "author": "Qonto",
```

Comment:
the original author is still the same, so I'm not a big fan of removing him here

Reply:
Ok, I wasn't sure about this. Will update.